tests/NHSD.BuyingCatalogue.Infrastructure.Tests/EnumeratorTests.cs

```diff
@@ -62,12 +61,10 @@ public void FromValueGivenValueSmallShouldBeEqual()
         [Test]
         public void FromValueGivenValueFourShouldThrow()
         {
-            var expectedValue = 4;
+            const int expectedValue = 4;
```

Comment:
Should this have a capitalised first letter?

Reply:
Not within a method, only instance level constants should start with a cap.

Reply:
That's fair, approved